tests/unit/test_onnx_parity_runner.py

```
@@ -0,0 +1,436 @@
+    assert str(rel) == "images/val2017/frame.jpg"
+    assert target == tmp_path.resolve() / "images" / "val2017" / "frame.jpg"
+
+
+def test_download_coco_person_keypoints_recovers_corrupt_zip(tmp_path, monkeypatch):
+    import vision_analysis_benchmark.onnx_parity as parity
+
+    zip_path = tmp_path / "annotations" / "annotations_trainval2017.zip"
+    zip_path.parent.mkdir()
+    zip_path.write_bytes(b"partial")
+
+    def fake_download(_url, target):
```

Ruff (0.15.17)

[warning] Missing return type annotation for private function `fake_download`

Add return type annotation: `None`

(ANN202)